app/views/home/index.html.haml

```diff
@@ -4,16 +4,54 @@
 
     .button-container
       = link_to user_mediawiki_omniauth_authorize_path, {class: 'button auth', method: :post} do
-        %i.icon.icon-wiki-logo
+        %i.icon.icon-wiki-purple
         = t('home.log_in')
 
-      = link_to user_mediawiki_signup_omniauth_authorize_path, {class: 'button button-inverse auth signup', method: :post} do
-        %i.icon.icon-wiki-logo
+      = link_to("javascript:void(0);", { class: 'button button-inverse auth signup', method: :post, onclick: 'openModal()' }) do
+        %i.icon.icon-wiki-purple
         = t('home.sign_up')
 
       %span.help{data: { title: t('home.sign_up_help') } }
         ?
 
+%div{class: 'wizard active undefined', id: 'signupModal', style: 'display: none'}
```

Comment:
I think I would rather leave this out of the server-rendered one, and just limit it to the EnrollCard, as this feature is specifically intended for students who are in a Wiki Education student program course, and few students sign up via the home page.

Reply:
i have discarded the changes for this page